examples/wearable/UIComponents/contents/controls/marquee/index.html

```diff
@@ -24,17 +24,14 @@ <h2 class="ui-title">
 			</h2>
 		</header>
 		<div class="ui-content">
-			<div class="ui-marquee ui-marquee-gradient" data-iteration="10" data-marquee-style="slide" id="marquee">
-				MarqueeTEST TEST message TEST for marquee
-			</div>
-			<div class="ui-marquee ui-marquee-ellipsis" data-delay="1000" data-marquee-style="scroll" id="marquee2">
```

Comment:
It would be good to have an example with **ui-marquee-ellipsis** if it's still supported.